dydx/dydxPresenters/dydxPresenters/_v4/Markets/Shared/SharedMarketPresenter.swift

```diff
@@ -64,10 +63,15 @@ class SharedMarketPresenter: HostedViewPresenter<SharedMarketViewModel>, SharedM
                                                                     sign: market.priceChange24HPercent?.doubleValue ?? 0 >= 0 ? .plus : .minus,
                                                                     coloringOption: .allText)
         }
-        if let key = asset?.resources?.primaryDescriptionKey {
+        // sometimes the descriptions are unavailable, need to check localized output to ensure availability
+        if let key = asset?.resources?.primaryDescriptionKey,
+           DataLocalizer.localize(path: "APP.\(key)") != "APP.\(key)"
+        {
```

Comment:
prevents this:
![IMG_4255939EAA06-1](https://github.com/user-attachments/assets/6f8a6d84-fc34-4d16-acd7-ac2e8e974dfd)